src/main/java/core/org/akaza/openclinica/bean/extract/odm/ClinicalDataReportBean.java

```diff
@@ -215,10 +215,22 @@ public void addNodeClinicalData(boolean header, boolean footer) {
                             xml.append("\" OpenClinica:EndDate=\"" + StringEscapeUtils.escapeXml(endDate));
                         }
 
+                        if (BooleanUtils.isTrue(se.getRemoved()))
+                            xml.append("\" OpenClinica:Status=\"" + StringEscapeUtils.escapeXml("removed"));
+                        else if (BooleanUtils.isTrue(se.getArchived()))
+                            xml.append("\" OpenClinica:Status=\"" + StringEscapeUtils.escapeXml("auto-removed"));
+                        else if (BooleanUtils.isTrue(se.getLocked()))
+                            xml.append("\" OpenClinica:Status=\"" + StringEscapeUtils.escapeXml("locked"));
+                        else if (BooleanUtils.isTrue(se.getSigned()))
+                            xml.append("\" OpenClinica:Status=\"" + StringEscapeUtils.escapeXml("signed"));
+                        else if (se.getWorkflowStatus() != null)
+                            xml.append("\" OpenClinica:Status=\"" + StringEscapeUtils.escapeXml(se.getWorkflowStatus().getDisplayValue()));
+
+
                         if(se.getWorkflowStatus()!=null) {
                             StudyEventWorkflowStatusEnum workflow = se.getWorkflowStatus();
                             if (!StringUtils.isEmpty(workflow)) {
-                                xml.append("\" OpenClinica:Status=\"" + StringEscapeUtils.escapeXml( workflow.getDisplayValue()));
+                                xml.append("\" OpenClinica:Workflow_Status=\"" + StringEscapeUtils.escapeXml( workflow.getDisplayValue()));
```

Comment:
rename this to "OpenClinica:WorkflowStatus" to be consistent with other attributes.

Reply:
done, renamed them